geomesa-accumulo/geomesa-accumulo-dist/src/main/assemblies/component.xml

```diff
@@ -233,6 +233,10 @@
             <destName>README-process.md</destName>
             <outputDirectory>docs</outputDirectory>
         </file>
+        <file>
```

Comment:
can you get this included in the deps section up top instead of the files? i think if you add it to the pom for the dist then use exclusion rules i think that's better for built artifacts.